.travis.yml

```diff
@@ -26,7 +26,8 @@ install:
 - npm run vscode:prepublish
 
 script:
-- npm test --silent
+# VS Code 1.36.1 fails to start on Linux for an unknown reason.
+- if [[ "$TRAVIS_OS_NAME" != "linux" ]]; then npm test --silent; fi
```

Comment:
This looks dodgy. Basically skipping the tests for linux

Reply:
I reenabled tests on Linux, seems to work, not sure why it wasn't working earlier.